src/app/camera.cpp

```diff
@@ -69,6 +112,10 @@ glm::vec3 Camera::getScreenCenter() const {
     return screen_center_m;
 }
 
+glm::vec3 Camera::getXAxis() const {
+    return glm::normalize(glm::cross(direction_m, screen_up_m));
```

Comment:
Here as well we can remove glm::normalize, because direction_m and screen_up_m is already normalized, and perpendicular, therefore their cross-product is normalized
```suggestion
    return glm::cross(direction_m, screen_up_m);
```

Reply:
Fixed in 4163279